dataactvalidator/config/sqlrules/a18_cross_file.sql

```diff
@@ -1,10 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.gross_outlay_amount_by_tas_cpe,
-    SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
-FROM appropriation approp JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
+	approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code,
+	approp.gross_outlay_amount_by_tas_cpe,
+	SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
+FROM appropriation AS approp
+	JOIN object_class_program_activity op
+		ON approp.tas = op.tas
+			AND approp.submission_id = op.submission_id
 WHERE approp.submission_id = {}
-GROUP BY approp.row_number, approp.tas, approp.gross_outlay_amount_by_tas_cpe
+GROUP BY approp.row_number,
+	approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code,
+	approp.gross_outlay_amount_by_tas_cpe
 HAVING approp.gross_outlay_amount_by_tas_cpe <> SUM(op.gross_outlay_amount_by_pro_cpe)
```

Comment:
Curious -- should this be 
```sql
 HAVING approp.gross_outlay_amount_by_tas_cpe <> gross_outlay_amount_by_pro_cpe_sum
```
They're equivalent, but one might describe the relationships that are being compared better.